blog/tech-talk-window-resize-behavior.md

```diff
@@ -0,0 +1,290 @@
+---
+title: 'Tech Talk #1: Improving Window Resize Behavior'
+date: 2025-12-19T00:00:00.000Z
+authors:
+  - name: nikwen
+    url: 'https://github.com/nikwen'
+    image_url: 'https://github.com/nikwen.png?size=96'
+slug: tech-talk-window-resize-behavior
+tags: [techtalk, internals]
+---
+
+_We're launching a new blog post series where we share glimpses into our work on Electron. If you find this work interesting, please consider [contributing](https://github.com/electron/electron/)!_
+
+---
+
+Recently, I worked on improving Electron and Chromium's window resize behavior.
+
+## The bug
+
+We were seeing an issue on Windows where old frames would become visible while resizing a window:
+
+![Animated GIF showing the issue where old frames would be shown while resizing windows](/assets/img/blog/tech-talk-window-resize-behavior/window-resize-bug-showing-old-frames.gif)
+
+## What made this bug particularly interesting?
+
+1. It was challenging.
+2. As you'll see later, there were two different bugs under the hood.
+
+## Fixing the bug
+
+With a bug like this, the first challenge is figuring out where to start looking.
+
+Electron builds upon Chromium. Chromium is the open source version of Google Chrome. When compiling Electron, Electron's source code is added into the Chromium source tree as a subdirectory. Electron then relies on Chromium's code to provide most of the functionality of a modern browser.
```

Comment:
nit: joining two clauses together to flow a bit better

```suggestion
Electron builds upon Chromium, the open source version of Google Chrome. When compiling Electron, Electron's source code is added into the Chromium source tree as a subdirectory. Electron then relies on Chromium's code to provide most of the functionality of a modern browser.
```

Reply:
Thanks for the suggestion! I'd prefer not to change this. I prefer shorter sentences.